python/core/auto_generated/qgsrange.sip.in

```diff
@@ -241,6 +241,19 @@ Returns ``True`` if this range contains a specified ``element``.
     bool overlaps( const QgsTemporalRange<T> &other ) const;
 %Docstring
 Returns ``True`` if this range overlaps another range.
+%End
+
+    bool extend( const QgsTemporalRange<T> &other );
```

Comment:
I find the docs confusing here - it's not clear what happens when a disjoint the is specified. Maybe instead of "by adding an other range" it should say "by extending this range out to include an other range".